pages/index.tsx

```diff
@@ -99,12 +99,12 @@ export default function Home() {
               alt="Dashed eye representing data privacy."
             />
             <div className="mt-4 mb-2 text-black text-xl font-bold">Distributed network</div>
-            <div>Connecting objects, people and organizations while ensuring independance.</div>
+            <div>Objects, people and organizations are interconnected but remain self-reliant.</div>
           </div>
           <div className="mb-16">
             <Image width={60} height={60} src={merge} alt="Dashed eye representing data privacy." />
             <div className="mt-4 mb-2 text-black text-xl font-bold">Performant editing</div>
-            <div>Excelling at data synchronization using a dynamic data structure.</div>
+            <div>Data synchronisation is optimized by using efficient dynamic data structures.</div>
```

Comment:
optimized + efficient has the same meaning, we can remove efficient in my opinion

Reply:
Hum, justement efficience is the explanation why we are optimized. dynamic data structures are not optimized by design.